microsoft-edge/progressive-web-apps/demo-pwas.md

```diff
@@ -140,6 +176,38 @@ BPM Techno uses the following features:
 | Window Controls Overlay | 1DIV uses the space normally reserved by the title bar to display a logo, search field, and button. | [Display content in the title bar area using Window Controls Overlay](how-to/window-controls-overlay.md) |
 
 
+<!-- ====================================================================== -->
+## Temperature converter
+
+A simple PWA demo app that converts temperatures.  Used for [Get started with PWAs](./how-to/index.md).
+
+[App](https://microsoftedge.github.io/Demos/pwa-getting-started/), [source code and Readme file](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-getting-started).
+
+
+<!-- ====================================================================== -->
+## Email client
+
+A simulated email client PWA that demonstrates how to use PWA protocol handlers.    
+
+[App](https://microsoftedge.github.io/Demos/email-client/), [source code and Readme file](https://github.com/MicrosoftEdge/Demos/tree/main/email-client/).
+
+
+<!-- ====================================================================== -->
+## Application Title Meta Tag
+
+A demo web app to showcase the `application-title` meta tag.
+
+[App](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-application-title), [source code and Readme file](https://microsoftedge.github.io/Demos/pwa-application-title/).
+
+
+<!-- ====================================================================== -->
+## Edge demos (/pwa-pwastore) (Web Install API and CSS Masonry)
```

Comment:
name of demo?  alts; key words:
* Edge demos
* /pwa-pwastore/
* Web Install API
* CSS Masonry

@captainbrosset